test/functional/l2adjacent/test_CLUDS.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from f5.bigip import ManagementRoot
+from f5.utils.testutils.registrytools import register_device
+from pprint import pprint as pp
+import sys
+import time
+import pytest
+import yaml
+
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+CREATIONURISET = set( [u'https://localhost/mgmt/tm/ltm/snat-translation/~Project_128a63ef33bc4cf891d684fad58e7f2d~snat-traffic-group-local-only-ce69e293-56e7-43b8-b51c-01b91d66af20_0?ver=11.6.0', u'https://localhost/mgmt/tm/ltm/snatpool/~Project_128a63ef33bc4cf891d684fad58e7f2d~Project_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.0', u'https://localhost/mgmt/tm/net/fdb/tunnel/~Project_128a63ef33bc4cf891d684fad58e7f2d~tunnel-vxlan-45?ver=11.5.0', u'https://localhost/mgmt/tm/net/route-domain/~Project_128a63ef33bc4cf891d684fad58e7f2d~Project_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.0', u'https://localhost/mgmt/tm/net/self/~Project_128a63ef33bc4cf891d684fad58e7f2d~local-bigip1-ce69e293-56e7-43b8-b51c-01b91d66af20?ver=11.6.0', u'https://localhost/mgmt/tm/net/tunnels/tunnel/~Project_128a63ef33bc4cf891d684fad58e7f2d~tunnel-vxlan-45?ver=11.6.0', u'https://localhost/mgmt/tm/sys/folder/~Project_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.0'])
```

Comment:
Move this "exact" string into a README to capture as an artifact, but need to generalize this to learn the tenant_id from the deployment on the fly so make this a repeatable test for anyone to run in their session.